config/settings.yml

```diff
@@ -353,11 +353,14 @@ claims_api:
   disability_claims_mock_override: false
   schema_dir: config/schemas
 
-# Settings for Redis
-# TODO(knkski): Move all redis settings here?
 redis:
   host: localhost
   port: 6379
+  app_data:
+    url: redis://localhost:6379
+    # secondary_url: localhost:6378
```

Comment:
Still has the wrong port number, right? and needs the `redis://`

Reply:
Updated the `redis://` protocol prefix.

The host and port aren't "wrong" per-se (since it's commented out), and I dont think this should ever be defined in `settings.yml` since it's overidden in the various prod, staging, dev `settings.local.yml` files.

Reply:
I'm open to removing it altogether.

I left it in because of how we removed the enabled/disabled flag. W/o that flag and/or this comment, a local developer would have no way of knowing that they can define `redis.app_data.secondary_url` to enable redis duplication.

Reply:
Could also update the value in the comment to something very obviously wrong, like:
```YAML
secondary_url: redis://fake_secondary_redis:9999
```

I've followed this pattern in the past since it makes things very clear & searchable should a developer accidentally run into it.